vendor/bat-native-ads/include/bat/ads/ad_content_info.h

```diff
@@ -26,13 +26,13 @@ struct ADS_EXPORT AdContentInfo final {
   bool operator==(const AdContentInfo& rhs) const;
   bool operator!=(const AdContentInfo& rhs) const;
 
-  AdContentLikeActionType ToggleThumbUpActionType() const;
-  AdContentLikeActionType ToggleThumbDownActionType() const;
+  [[nodiscard]] AdContentLikeActionType ToggleThumbUpActionType() const;
```

Comment:
As I understand, [[nodiscard]] should be used in specific cases.
https://groups.google.com/a/chromium.org/g/chromium-dev/c/Z9wRGFoYrtU

> Only use when failing to handle the return is definitely a problem. 

> such as returning ownership of a raw pointer to a resource, or an error code.

Reply:
See https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/c++-features.md#nodiscard-attribute-allowed

Reply:
yeah, there is a note there:

> This replaces the previous WARN_UNUSED_RESULT macro, which was a wrapper around the compiler-specific __attribute__((warn_unused_result)).

Reply:
Remove [[nodiscard]], thanks